src/compute/src/render/join/delta_join.rs

```diff
@@ -142,7 +148,7 @@ where
                     let update_stream = match val {
                         Ok(local) => {
                             let arranged = local.enter_region(region);
-                            let (update_stream, err_stream) = build_update_stream(
+                            let (update_stream, err_stream) = dispatch_update_stream_local(
```

Comment:
There is this pattern where you produce the dispatch method name by replacing the first verb in the target method with `dispatch`. Not a huge deal, but it does change the meaning a bit. For example:

* `dispatch_finished_data` makes it sound like we are dispatching the data somehow, when really we are dispatching the _collection_ of the data.
* `dispatch_update_stream` makes it sound like we are dispatching the updating of a stream, when really we are dispatching the building of an update stream.

Reply:
That's a good point; I made the naming less confusing by including the verb also from the target methods in the names.